### DISCUSSION TITLE
The struggle is real!

### DISCUSSION THREAD

**Discussion by kassyoveson** [USU-ITLS-x265-23-summer/class-repo#31] [Q&A]
I have honestly struggled a lot this week with learning github. It is still a learning experience. I struggled with Task 3. As I attempted to follow some steps, I was unable to find icons, name files correctly, etc. However, in this struggle, I was able to figure out how to restart a task, so I could try again. I don't know exactly what I did to reset it. I think it was through the task page and you go to web description. Anyway, this is getting long. Next week, I will try to post questions as I go through the assignment. All I know is that the struggle was real for me this week and I don't think I am alone.

**Comment by chrisbabits** [comment]
I agree that the struggle's been real with GitHub! You are definitely not alone, @kassyoveson! #solidarity

**Comment by metasean** [comment]
Sadly, git in general is a challenge and has something of a stepped learning curve.  It starts out really hard, but once you've been through the process a few times, it levels out, until you need to learn a new aspect of it, then it's challenging again, until you finally get it.  There are a lot of levels!.

The good news is that HTML, CSS, & JavaScript are much easier to learn!  There's a ***lot*** to learn, and transferring that knowledge to your project is challenging for some learners, but I'm hoping you will have learned enough by then, that you know when and how to ask questions so you can keep rolling —if not, that's what office hours are for, and if there isn't an office hours in the next few days for you to attend, please ping me and I'll try to find a time that we can work through the challenge together.

**Comment by coopsand** [comment]
I've used Git in the past, but I just want to let you know that you are not alone. It has certainly been a struggle for me as well. Relearning the steps and how it has organized has been very frustrating for me.
